spec/models/author_spec.rb

```diff
@@ -19,4 +19,12 @@
                                 "book #2", "book #3", "book #4")
     end
   end
+
+  it 'sets a slug when saved do' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.